cli/lib/kontena_cli.rb

```diff
@@ -87,6 +115,26 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    if log_target.respond_to?(:tty?) && log_target.tty?
+      logger = Logger.new(log_target)
+      logger.datetime_format = "%M%S.%2N "
```

Comment:
The full timestamp is so long. I know or don't care what day and time it is when running CLI commands. The log file gets full timestamps.

The `DEBUG` duplicates `D`. The PID is 99% irrelevant and when it is relevant, you could see it in the log file.